lib/graph-script.js

```diff
@@ -46,6 +47,7 @@ function node (state, createEdge) {
   b.transform(yoyoify, { global: true })
   b.transform(envify, { global: true })
   b.transform(uglifyify, uglifyOpts(entry))
+  b.plugin(commonShake)
```

Comment:
Shouldn't this be called pre-uglify? If I recall correctly, Indutny's tree shaker only comments code.

Reply:
oh yeah that's a good catch, it needs to be uglified afterward--uglifyify runs as a transform though, which always happens _before_ the plugin runs. so the final bundle should be passed thru uglify in order to actually clean the dead code bits.

Reply:
Fixed!